recipe/meta.yaml

```
@@ -17,10 +17,10 @@ build:
 
 requirements:
   host:
-    - python >=3.9,<4.0
+    - python {{ python_min }}
```

```suggestion
    - python {{ python_min }}
    - poetry
```

I don't like poetry :) Why not hatchling or setuptools or uv?